diff -r aaa3eec421aa autobuild.xml
--- a/autobuild.xml	Wed Mar 23 22:10:03 2011 +0000
+++ b/autobuild.xml	Sat Mar 16 18:20:35 2013 -0700
@@ -117,7 +117,7 @@
         </map>
       </map>
       <key>version</key>
-      <string>8c</string>
+      <string>9</string>
     </map>
     <key>type</key>
     <string>autobuild</string>
diff -r aaa3eec421aa build-cmd.sh
--- a/build-cmd.sh	Wed Mar 23 22:10:03 2011 +0000
+++ b/build-cmd.sh	Sat Mar 16 18:20:35 2013 -0700
@@ -7,7 +7,8 @@
 # make errors fatal
 set -e
 
-JPEGLIB_VERSION="8c"
+JPEGLIB_VERSION="9"
+JPEGLIB_MD5="??"
 JPEGLIB_SOURCE_DIR="jpeg-$JPEGLIB_VERSION"
 
 if [ -z "$AUTOBUILD" ] ; then 
@@ -23,16 +24,39 @@
 eval "$("$AUTOBUILD" source_environment)"
 set -x
 
+# get it
+#
+case "$AUTOBUILD_PLATFORM" in
+    "windows")
+       	JPEGLIB_ARCHIVE="jpegsr$JPEGLIB_VERSION.zip"
+       	JPEGLIB_MD5="a41286ae0654b47fda735d24e9a71f3d"
+	JPEGLIB_URL="http://www.ijg.org/files/$JPEGLIB_ARCHIVE"
+	#fetch_archive "$JPEGLIB_URL" "$JPEGLIB_ARCHIVE" "$JPEGLIB_MD5"
+	unzip "$JPEGLIB_ARCHIVE"
+      ;;
+     "darwin")
+       	JPEGLIB_ARCHIVE="jpegsrc.v$JPEGLIB_VERSION.tar.gz"
+      	JPEGLIB_MD5="b397211ddfd506b92cd5e02a22ac924d"
+	JPEGLIB_URL="http://www.ijg.org/files/$JPEGLIB_ARCHIVE"
+	fetch_archive "$JPEGLIB_URL" "$JPEGLIB_ARCHIVE" "$JPEGLIB_MD5"
+	extract "$JPEGLIB_ARCHIVE"
+       ;;
+      "linux")
+       	JPEGLIB_ARCHIVE="jpegsrc.v$JPEGLIB_VERSION.tar.gz"
+       	JPEGLIB_MD5="b397211ddfd506b92cd5e02a22ac924d"
+	JPEGLIB_URL="http://www.ijg.org/files/$JPEGLIB_ARCHIVE"
+	fetch_archive "$JPEGLIB_URL" "$JPEGLIB_ARCHIVE" "$JPEGLIB_MD5"
+	extract "$JPEGLIB_ARCHIVE"
+      ;;
+esac
+
 stage="$(pwd)/stage"
 pushd "$JPEGLIB_SOURCE_DIR"
     case "$AUTOBUILD_PLATFORM" in
         "windows")
             load_vsvars
-            
             nmake -f makefile.vc setup-v10
-            
             build_sln "jpeg.sln" "Release|Win32" 
-
             mkdir -p "$stage/lib/debug"
             mkdir -p "$stage/lib/release"
             cp "Release\jpeg.lib" \
@@ -45,8 +69,8 @@
             ./configure --prefix="$stage"
             make
             make install
-			mkdir -p "$stage/include/jpeglib"
-			mv "$stage/include/"*.h "$stage/include/jpeglib/"
+		mkdir -p "$stage/include/jpeglib"
+		mv "$stage/include/"*.h "$stage/include/jpeglib/"
         ;;
         "linux")
             CFLAGS="-m32" CXXFLAGS="-m32" ./configure --prefix="$stage"
